data/cache/cache_key.py

```diff
@@ -9,48 +9,57 @@ class CacheKey(namedtuple("CacheKey", ["key", "expiration"])):
     pass
 
 
-def for_repository_blob(namespace_name, repo_name, digest, version):
+def for_repository_blob(namespace_name, repo_name, digest, version, cache_config):
```

Comment:
Is it better here to pass the whole config here or have the callers pass the value as an int or valid durations string?

Reply:
I think it's better to have TTL. I'll make this change

Reply:
Right now I just pass `model_cache.cache_config` but if we are to pass TTL the caller should do something like 

`model_cache.cache_config.get('repository_blob_cache_ttl', 'default-value')` Here the caller has to know the key and also what the default value should be which shouldn't really be the caller's concern